index.mjs

```diff
@@ -0,0 +1,430 @@
+/*
+ * Copyright (c) 2021-2024 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { promises as fs } from 'node:fs';
+
+import { fixupConfigRules } from '@eslint/compat';
+import checkdigit from '@checkdigit/eslint-plugin';
+import ts from 'typescript-eslint';
+import sonarjs from 'eslint-plugin-sonarjs';
+import importPlugin from 'eslint-plugin-import';
+import noOnlyTests from 'eslint-plugin-no-only-tests';
+import noSecrets from 'eslint-plugin-no-secrets';
+import n from 'eslint-plugin-n';
+import tsParser from '@typescript-eslint/parser';
+import jest from 'eslint-plugin-jest';
+import js from '@eslint/js';
+import prettier from 'eslint-config-prettier';
+import { FlatCompat } from '@eslint/eslintrc';
+import unicorn from 'eslint-plugin-unicorn';
+
+const ignores = [
+  ...(await fs.readFile('.gitignore', 'utf-8')).split('\n').filter((path) => path.trim() !== ''),
```

Comment:
are we sure this is reading the files from the correct place?  i.e. from the project root, not from eslint-config root

Reply:
yes, tested and confirmed that it's reading the file belonging to the current project instead of eslint-config module.